static/app/components/button.tsx

```diff
@@ -193,6 +195,16 @@ type LinkButtonProps =
   | HrefLinkButtonPropsWithChildren
   | HrefLinkButtonPropsWithAriaLabel;
 
+/**
+ * Default sizes to use for SVGIcon
+ */
+const ICON_SIZES = {
+  zero: 'xs',
```

Comment:
actually maybe we want to let this be whatever the icon default is

Reply:
ooh, maybe we need to identify any places that exist already where the icon has no size specified, and the button is a bigger/smaller size... because after merging those cases would change (to be better i suppose)

or, ehh, they'll become better.

Reply:
I'll remove the `zero` here and let it not have a default